packages/prism-theme-odyssey/src/theme.scss

```diff
@@ -4,22 +4,22 @@
 
 :root {
   --font-size: #{$size-body-base};
-  --inlineCodeColor: #{cv('purple', '500')};
+  --inlineCodeColor: #{cv('blue', '000')};
   --inlineCodeBackground: #{cv('red', '300')};
-  --blockBackground: #{cv('gray', '000')};
-  --baseColor: #{cv('gray', '900')};
-  --selectedColor: #{cv('blue', '300')};
-  --highlightBackground:  #{cv('yellow', '000')};
-  --highlightAccent:  #{cv('yellow', '300')};
+  --blockBackground: #{cv('gray', '900')};
+  --baseColor: #{cv('blue', '000')};
+  --selectedColor: #{cv('turquoise', '500')};
+  --highlightBackground: #{cv('yellow', '000')};
+  --highlightAccent: #{cv('yellow', '300')};
   --commentColor: #{cv('gray', '500')};
-  --punctuationColor: #{cv('gray', '200')};
-  --propertyColor: #{cv('purple', '500')};
-  --selectorColor: #{cv('turquoise', '500')};
-  --operatorColor: #{cv('yellow', '900')};
+  --punctuationColor: #{cv('blue', '000')};
+  --propertyColor: #{cv('turquoise', '400')};
+  --selectorColor: #a0dcc3;
```

Comment:
Can we add this to ODS, even if it's not in general use, to maintain a SSoT?

Reply:
This was hardcoded because the color is part of the expanded palette which is not in Odyssey at the moment.
  
Once [OKTA-379938](https://oktainc.atlassian.net/browse/OKTA-379938) is complete, I will make the appropriate updates.

I have logged a ticket for this here:  [OKTA-383322](https://oktainc.atlassian.net/browse/OKTA-383322)

Reply:
works for me; thanks!